src/Utils.h

```diff
@@ -1,7 +1,32 @@
+#pragma once
+
 void ltrim(std::string& s);
 void rtrim(std::string& s);
 void trim(std::string& s);
 
 spdlog::sink_ptr CreateCustomSinkST(std::function<void(const std::string&)> aSinkItHandler, std::function<void()> aFlushHandler = nullptr);
 spdlog::sink_ptr CreateCustomSinkMT(std::function<void(const std::string&)> aSinkItHandler, std::function<void()> aFlushHandler = nullptr);
-std::shared_ptr<spdlog::logger> CreateLogger(const std::filesystem::path& aPath, const std::string& aID, spdlog::sink_ptr aExtraSink = nullptr, const std::string& aPattern = "[%Y-%m-%d %H:%M:%S UTC%z] [%l] %v");
\ No newline at end of file
+std::shared_ptr<spdlog::logger> CreateLogger(const std::filesystem::path& aPath, const std::string& aID, spdlog::sink_ptr aExtraSink = nullptr, const std::string& aPattern = "[%Y-%m-%d %H:%M:%S UTC%z] [%l] %v");
+
+template <typename ...Args>
+void TryLuaFunction(std::shared_ptr<spdlog::logger> logger, sol::function func, Args... args)
```

Comment:
This shouldn't be in Utils I think

Reply:
Moved into ScriptContext.cpp, as that was only place it was used in the end (wanted to use it elsewhere too before but found out that is unusable in other cases....)
Hope it is in better place now :P